src/index.js

```diff
@@ -53,51 +59,58 @@ const run = async () => {
             }
 
             const changelogContent = await readFile(`${folder}CHANGELOG.md`, {encoding: 'utf-8'});
-            const {
-                isUnreleased,
-                version,
-                date,
-                skeleton,
-            } = validateChangelog(changelogContent);
-
-            // Checks if the branch is release or once of release_branches input.
-            if (releaseBranches.find((releaseBranch) => branch.startsWith(releaseBranch))) {
-                if (isUnreleased) {
-                    throw new Error(`"${branch}" branch can't be unreleased`);
-                }
-
-                if (!version || version === 'Unreleased') {
-                    throw new Error(`"${branch}" branch should have a version`);
-                }
-
-                if (!date) {
-                    throw new Error(`"${branch}" branch should have a date`);
-                }
-
-                const {version: packageVersion} = JSON.parse(await readFile(`${folder}package.json`, {encoding: 'utf-8'}));
-                if (packageVersion !== version) {
-                    throw new Error(`The package version "${packageVersion}" does not match the newest version "${version}"`);
-                }
+            validateChangelog(changelogContent);
+        }
 
-                const packageLockStats = await stat(`${folder}package-lock.json`);
+        // Checks if the branch is release or once of release_branches input.
+        if (releaseBranches.find((releaseBranch) => branch.startsWith(releaseBranch))) {
+            const changelogs = modifiedFiles.filter((file) => file.endsWith('CHANGELOG.md'));
 
-                if (packageLockStats) {
-                    const {version: packageLockVersion} = JSON.parse(await readFile(`${folder}package-lock.json`, {encoding: 'utf-8'}));
-                    if (packageLockVersion !== version) {
-                        throw new Error(`The package-lock version "${packageVersion}" does not match the newest version "${version}"`);
-                    }
-                }
+            /**
+             * If only one changelog is modified
+             * there is no doubt of which one is correct
+             */
+            if (changelogs.length === 1) {
```

Comment:
But we are talking about release branch, in this case at least one must be modified.
And it case is not the correct one it should fail in the rest of the checks.

Reply:
let's say I didn't modified the Changelog of SEO but I created a release Branch and PR to production, it might be that other changelogs are modified (e.g. App, embed) so It won't fail because there are changelogs that are being updated but these are not the one that we interested in.

Reply:
This check only happens when only 1  changelog is modified, in this example you are modifying more than one, so it won't go to this check.

But lest's say that we had only 1 previous modified changelog, you could trigger this check only if you did not modify at all the changelog for SEO, but why make a release branch if you don't make any change?

Reply:
Isn't that why we have The changelog action? to validate this?